recipes/frugen/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+from conan import ConanFile
```

Comment:
```suggestion
from conan import ConanFile
from conan.errors import ConanInvalidConfiguration
```

Reply:
done

Reply:
Now it passes build, thank you!
But I still don't understand, why it did not work with windows runner, will try again.